dev-env.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/sh
+
+## Run docker Compose
+sudo docker compose --env-file backend/.env up -d
+
+## change directory to backend
+cd backend
+
+## Run migration
+npm run migrate:dev
```

Comment:
Dev script should just start the server and docker container at both frontend and backend.